### PR TITLE
updated text and buttons in snackbar

### PR DESCRIPTION
**Pull request by marionleborgne**
Incorporating Jeff's feedback regarding the snackbar.

### CODE REVIEW

**Comment by marionleborgne** [comment]
@lscheinkman please review when you have time. Thanks

File: unicorn/app/config/default.json [hunk 1]
```diff
@@ -52,8 +52,7 @@
   },
   "snackbar": {
     "completed": {
-      "message": "HTM Model completed for %s",
-      "action" : "Results"
```
Comment (lscheinkman) [inline unicorn/app/config/default.json]: You should also remove the action handler code from the snackbar.
https://github.com/marionleborgne/numenta-apps/blob/snackbar-with-jeff-feedback/unicorn/app/browser/components/Model.jsx#L476-L477

**Comment by marionleborgne** [comment]
Thanks @lscheinkman. I made the update.

**Comment by lscheinkman** [comment]
👍 LGTM

**Pull request merged**